testing/helpers.go

```diff
@@ -0,0 +1,65 @@
+package testing
+
+import (
+	"fmt"
+
+	"github.com/ipfs/go-datastore"
+	"github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/libp2p/hydra-booster/node"
+	"github.com/libp2p/hydra-booster/node/opts"
+)
+
+var defaults = []opts.Option{opts.Datastore(datastore.NewMapDatastore()), opts.BootstrapPeers(nil)}
+
+// SpawnNode a new Hydra nodes with an in memory datastore and 0 bootstrap peers by default.
+// It also waits for bootstrapping to complete.
+func SpawnNode(options ...opts.Option) (*node.HydraNode, error) {
+	nd, bsCh, err := node.NewHydraNode(append(defaults, options...)...)
+	if err != nil {
+		return nil, err
+	}
+
+	for {
+		status, ok := <-bsCh
```

Comment:
Asked to the group if for single notifications, if the best is channels or waitgroups. Not blocking the PR. will soon learn from the maaastersss :)